src/toxcore/dht/server.rs

```diff
@@ -896,26 +900,6 @@ mod tests {
             }
         }).collect().wait().unwrap();
     }
-    // send_ping_req()
-    #[test]
```

Comment:
Why did you remove the test?

Reply:
send_ping_req() is moved into send_pings() to avoid deadlock

Reply:
OK